src/components/Books.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <ul className="Book-list">
+      <li>
+        <h2>Book 1</h2>
+        <button type="button">Remove</button>
+      </li>
```

Comment:
- [ ] The Books component should render a list of books using the `Book` component
- [ ] TheBook component should take the book data as a prop and render the title and the author

This will be much more easier to render so many books

Reply:
Done